src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -336,6 +336,11 @@ <h1 flex class="md-title kd-replicasetdetail-app-name">
           </tr>
           </tbody>
         </table>
+      </md-content>
+      <md-content class="kd-replicasetdetail-no-events" ng-if="!ctrl.events">
+        <md-icon class="material-icons kd-replicasetdetail-no-events-icon">info_outline</md-icon>
+        <div class="kd-replicasetdetail-no-events-text">No events were found</div>
```

Comment:
I can't find any information in documentation about event expiration. Could you provide me some link?

About message. Should it be some tooltip on icon or just plain text? Something similar to help section on deploy page?